data/assets/scene/digitaluniverse/stars.asset

```diff
@@ -132,8 +126,8 @@ asset.export(SunStar)
 asset.meta = {
   Name = "Stars",
   Version = "3.0",
```

Comment:
```suggestion
  Version = "4.0",
```

but @alexanderbock do we need this update?  Or is the update you did with the star renderable update enough?